### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by arjundevn**
login

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sloshed</title>
```
Comment (ganesh-kanchi) [inline login.html:7]: Title could be changed to "Login to Sloshed".

File: wishlist.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sloshed</title>
```
Comment (ganesh-kanchi) [inline wishlist.html:7]: can be changed to "Wishlist | Sloshed"

**Review by ganesh-kanchi — APPROVED** [review]
look pretty good to be merged.

File: login.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+</head>
+<body>
+    <div class="wrapper-products">
+        <div class="header">
+            <div class="brand">
+                <a href="/index.html"><h2>Sloshed.</h2></a>
+                <p>Get your booze before you snooze</p>
+                <p>10 mins delivery of your favourite alcohol</p>
+            </div>
+            <input type="text" class="search-bar" cols="20" rows="2" placeholder="🔍Search"></input>
```
Comment (Kothalkarpriya) [inline login.html:20]: here as well remove the closing tag for input

File: wishlist.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+</head>
+<body>
+    <div class="wrapper-products">
+        <div class="header">
+            <div class="brand">
+                <a href="/index.html"><h2>Sloshed.</h2></a>
+                <p>Get your booze before you snooze</p>
+                <p>10 mins delivery of your favourite alcohol</p>
+            </div>
+            <input type="text" class="search-bar" cols="20" rows="2" placeholder="🔍Search"></input>
```
Comment (Kothalkarpriya) [inline wishlist.html:20]: here as well

File: wishlist.html [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+                    <a href="/cart.html" class="fa fa-cart-shopping"></a>
+                    <span class="badge-icon-img  icon">2</span>
+                </div>
+            </div>
+        </div>
+        <div class="products-container">
+            <div class="products-listings">
+                <h1 class="products-title">Wishlist</h1>
+                <div class="card card-image">
+                    <img class="card-img" src="./assets/bira-card.jpg" alt="Card image">
```
Comment (Kothalkarpriya) [inline wishlist.html:37]: for image use / before closing the tag

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Look into my comments

File: wishlist.html (hunk 5 above)
Comment (arjundevn) [reply wishlist.html:37]: Are you sure this is a good practice? I checked MDN as well: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img

Comment (Kothalkarpriya) [reply wishlist.html:37]: As you head towards React you will get problems with this type of convention

**Pull request merged**